packages/canvas-tokens/utils/filters/index.ts

```diff
@@ -59,6 +59,13 @@ export const isBreakpoints: Matcher = token => {
 };
 
 export const filterCodeTokens: Matcher = token => {
-  const excludedTokens = ['level', 'shadow', 'typescale'];
+  console.log('token>>', token);
```

Comment:
Are the `console.log` needed for the build?

Reply:
no, I was just checking stuff, thanks for catching it!